### PR TITLE
Add Django snippet support

### PR DESCRIPTION
**Pull request by cryocaustik**
Using snippets from [VS Code Django extension](https://github.com/vscode-django/vscode-django). 

This would be very helpful when developing in Django with Jinja2 as the template engine.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -81,6 +81,13 @@
         "scopeName": "source.jinja",
         "path": "./syntaxes/jinja.tmLanguage.json"
       }
-    ]
+    ],
+    "snippets": [
```
Comment (samuelcolvin) [inline package.json]: could you use 2 spaces for indents here.

Comment (cryocaustik) [reply package.json]: Done. 
I also adjusted some of the snippets that were to be jinja2, while removing those that do not have an equivalent (e.g. Spaceless). I've tested all the remaining snippets and they are working well.

Comment (samuelcolvin) [reply package.json]: Maybe my view isn't updating, but this still seems to be wrongly indented?

Comment (cryocaustik) [reply package.json]: it is probably because the new commit is linked below this comment section; the indentation should be fixed in c24f369 :smile:

Comment (samuelcolvin) [reply package.json]: the indent isn't changed in c24f369.

I'll fix it myself and release.

Comment (samuelcolvin) [reply package.json]: Weird, the indentation was right but showing wrongly. Sorry.

New version released.

**Pull request merged**